homework-5/Homework-5.playground/Contents.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+var country: [String: String]
+country = [
+    "Germany": "Western Europe. The surname Hitler is banned in Germany",
+    "Italy": "Southern Europe. The ending of all words in Italian only with a vowel",
+    "France": "Western Europe. France is the birthplace of the bicycle",
+]
+
+country["Italy"] = nil
+country["Spain"] = "located on the Iberian Peninsula. Football is considered the national sport"
+country["Germany"] = "The first book was printed in Germany"
+country["France"] = "France is the birthplace of cinema"
+print(country)
+
+var city: [String]
+city = [
+    "Amsterdam",
+    "Copenhagen",
+    "Vilnius",
+]
+
+var nameCountru = (["Germany: Western Europe. The surname Hitler is banned in Germany", "Italy: Southern Europe. The ending of all words in Italian only with a vowel", "France: Western Europe. France is the birthplace of the bicycle"])
```

Comment:
We need to concat continent and fact programmatically from the initial dictionary because they should be as two separate elements and we need them as one string